Doc/library/math.rst

```diff
@@ -136,6 +136,15 @@ Number-theoretic and representation functions
    .. versionadded:: 3.5
 
 
+.. function:: lcm(a, b)
+
+   Return the least common multiple of integers *a* and *b*.  If either *a* or
+   *b* is nonzero, then the value of ``lcm(a, b)`` is the smallest positive 
+   integer that is divisible by both *a* and *b*.  ``lcm(0, 0)`` returns ``0``.
```

Comment:
Language nitpick: suggest "is a multiple of" rather than "is divisible by", to better tie in with the "least common multiple" name, and to avoid potential confusion about what it means to be divisible by 0.

Reply:
In fact, there's no need to state the definition in two separate parts. In *all* cases, the value of ``lcm(a, b)`` is the least nonnegative integer which is a multiple of both `a` and `b`. That's enough to cover corner cases involving zero and negative numbers all by itself, but it may still be worth having a second sentence clarifying that if either input is zero, the result is zero. (People get unduly worried about zero being special in some way.)